test/elixometer_test.exs

```diff
@@ -217,10 +221,22 @@ defmodule ElixometerTest do
     assert ns > 1_000_000
   end
 
-  test "a timer defined in the module's declaration" do
-    rv = DeclarativeTest.my_timed_method(1, 2, 3, 4)
+  test "a bodyless timer defined in a module raises a RuntimeError" do
+    on_exit(fn ->
+      :code.delete(BodylessModule)
+      :code.purge(BodylessModule)
+    end)
 
-    assert rv == 1 + 2 + 3 + 4
+    assert_raise RuntimeError, "timed function must have a body", fn ->
+      defmodule BodylessModule do
+        use Elixometer
+        @timed(key: :auto)
+        def bodyless
```

Comment:
And now I'm curious why our Travis builds aren't also running on the older version in the testing matrix, too.